### PR TITLE
Hearthstone not launching

### LINKED ISSUE #3708: Deck Tracker not Launching Hearthstone
### Expected Behavior
I expected to press the "Launch Hearthstone" button and have the Hearthstone app open with the deck tracker active.

### Actual Behavior
Nothing happened - the "Start Hearthstone" button was gray for a second but then returned to its original state without the Hearthstone app having been opened. 

### Steps to reproduce behavior
Just clicking on start Hearthstone.
I've tried the other steps for it not working, such as changing the path, but none of it has worked so far. -

### PR DESCRIPTION
**Pull request by CLJunge**
- [X] I have read the [Contributing Guidelines](https://github.com/HearthSim/Hearthstone-Deck-Tracker/blob/master/CONTRIBUTING.md#contributing)
- [X] I have already signed the CLA 

Resolves #3708 

Tested with standard and beta client.

### CODE REVIEW

File: Hearthstone Deck Tracker/Utility/HearthstoneRunner.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ internal class HearthstoneRunner
 						return;
 					}
 				}
-				Process.Start("battlenet://WTCG");
+				Process.Start($"{bnetProc?.MainModule.FileName}", "--exec=\"launch WTCG\"");
```
Comment (azeier) [inline Hearthstone Deck Tracker/Utility/HearthstoneRunner.cs]: The null check needs to happen outside of this.

**Comment by CLJunge** [comment]
Alright, changed it.

**Comment by CLJunge** [comment]
Added another error message in case the process isn't available.

**Comment by azeier** [comment]
Looks good. Thanks! I'll get a release out tomorrow

**Pull request merged**